back/app/services/xlsx_export/input_sheet_generator.rb

```diff
@@ -161,7 +161,7 @@ def author_report_fields
     def meta_report_fields
       [].tap do |meta_fields|
         meta_fields << created_at_report_field
-        meta_fields << published_at_report_field if participation_method.supports_publication?
+        meta_fields << published_at_report_field if participation_method.supports_public_visibility?
```

Comment:
I like this public / private distinction. Makes it very clear.

Reply:
I do too, but I'm wondering whether the `published_at` field isn't relevant for non-public things too. Arguably the name is confusing, but even if you post something that's not public that can first be in draft, there is still a moment of submissions that could be relevant. Maybe `submitted_at` would be more clear? (again, doesn't need to change now, just thinking out loud again)

Reply:
Agreed `submitted_at` would make more sense than `published_at` for survey responses, as they're not public. Similarly, I wonder if we could make the `publication_status` more granular in the future, also supporting a `submitted` status (which can be useful for proposals under review that should not be public yet). This would make it easy to support a `published` or `public` scope for inputs (@IvaKop and I were discussing today that the ideas endpoint should always return only public inputs as it can be accessed by all users).